AdyenActions/AdyenActionComponent.swift

```diff
@@ -114,10 +103,10 @@ public final class AdyenActionComponent: ActionComponent, Localizable {
     }
     
     private func handle(_ action: ThreeDS2ChallengeAction) {
-        guard let threeDS2Component = threeDS2Component else { return }
+        guard let threeDS2Component = currentActionComponent as? ThreeDS2Component else { return }
```

Comment:
and assertionFailure here? "ThreeDS2Component is nil, There must be a `ThreeDS2FingerprintAction` action preceding a `ThreeDS2ChallengeAction` action"

Reply:
missed this during cherry-picking, thanks!